test/integration/controllers/WebActionsController.test.js

```diff
@@ -40,12 +104,12 @@ describe('WebActionsController', () => {
           invitePhone1: null,
           invitePhone2: '',
           invitePhone3: undefined,
-        }
+        },
       };
 
       const data = WebActionsController.createMobileCommonsReferralRequest(mockRequest);
       assert.equal(data, false);
-    })
+    });
 
     it('should handle building the request for 1 friend referral', () => {
```

Comment:
3 referrals are required for the scholarship, but all 3 are not for the BTS campaign. So I didn't want to make that requirement here. For now, what's required - which currently is all - is defined in BetaSignUpForm. There's probably some extra work that would need to be done if that component's gonna be reused for BTS.